main.py

```diff
@@ -39,34 +41,27 @@ async def main():
 
     workflow_id = str(uuid.uuid4())
 
-    historian = create_filesystem_historian(
-        saved_state, 'demo', register_user
-    )
+    storage = LocalFileSystemBlobStorage(saved_state)
+    history = PersistentHistory(workflow_id, storage)
 
-    workflow_task = historian.run('Howdy')
-    await asyncio.sleep(0.1)
+    async with WorkflowManager(workflow_id, storage, lambda arg: history, lambda arg: register_user) as manager:
```

Comment:
```python
from src.quest import create_filesystem_manager
...
async with create_filesystem_manager(saved_state, 'demo', lambda arg: register_user) as manager:
```
Also, we can move line 42 to line 48.

Reply:
And of course delete lines 44 and 45.

Reply:
Maybe instead of hard-coding `'demo'` as the workflow manager namespace, have a variable called `workflow_namespace = 'demo'` and use that.

Reply:
Ah - I see that you use `'demo'` in other places. Maybe use `'namespace'` instead.

Reply:
Great, I've made a pushed these changes